### ISSUE TITLE
caskl return error 1

### ISSUE THREAD

**Issue by blogabe** [sorin-ionescu/prezto#1866]
### Description
`caskl` tries to run new way to list casks, but an error code is returned.

### Expected behavior
Listing of installed casks

### Actual behavior
```
Error: Calling brew cask list is deprecated! Use brew list --cask instead.
-> [1]
```

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

### Versions

  - Prezto commit: 0a405494cae54e208ff550c1ba83380ed115410a
  - ZSH version: zsh 5.7.1
  - OS information: macOS 10.15.6

**Comment by srijanshetty** [comment]
I was able to get a repro with latest `brew`. That being said, it's a 'Warning' and only on the latest version of `brew`. Considering that a lot of people will be using older versions of `brew`. I do not believe it will be prudent to change the existing alias. A possible upgrade path can be providing an alternate alias for `brew list --cask`.

**Comment by jeffwidman** [comment]
Brew auto-updates every so often, so it won't be _that_ long until people are on the newer version.

**Comment by indrajitr** [comment]
Closed in #1905.
